### PR TITLE
Reload reply iframe instead of removing it

### PR DESCRIPTION
**Pull request by evanhellmuth**
Fixes https://github.com/FlowCrypt/flowcrypt-browser/issues/1564

Note that this UX differs from that of deleting a normal gmail reply:
Normal reply --> delete draft --> editor closes and user needs to click reply again
Secure reply --> delete draft --> text disappears, any extra recipients disappear, but editor stays open.

This was the ask in https://github.com/FlowCrypt/flowcrypt-browser/issues/1564, and I think it's a fine UX, just wanted to make the difference explicit.

### CODE REVIEW

**Comment by tomholub** [comment]
Thanks for pointing out the difference! It's not ideal but it gets the job done without being horribly annoying to the user.

**Comment by tomholub** [comment]
Remember to use the `issue-1564-reply-securereply-delete-flow-fix` branch name format to have your changes tested by automated tests.

**Comment by evanhellmuth** [comment]
@tomholub after changes UI now displays the following screenshot after deleting a secure reply. Matches Gmail UI perfectly (really clear directions from you, thanks for that):
screen shot 2019-02-24 at 6 51 03 pm

File: extension/js/common/composer.ts [hunk 1]
```diff
@@ -299,9 +299,9 @@ export class Composer {
     }, this.getErrHandlers(`change footer`)));
     $('.delete_draft').click(Ui.event.handle(async () => {
       await this.draftDelete();
-
       // Reload iframe so we don't leave users without a reply UI.
-      window.location.reload();
+      this.v.skipClickPrompt = false;
+      window.location.href = Env.urlCreate(window.location.protocol + '//' + window.location.hostname + window.location.pathname, this.v);
```
Comment (tomholub) [inline extension/js/common/composer.ts]: You could use `window.location.href.split('?')[0]` which is shorter but your is cleaner so it's just a matter of taste. Looks good.

**Pull request merged**